### PR TITLE
RADFISH-222 Multi Step Form Updates

### PR DESCRIPTION
**Pull request by IsaiahHarris**
- Title Copy Update: “where the form needs to keep track of the current step that the user is on.”
- Title Copy Update: “Multi-Step”
- “Begin Form” Button should be centered in the middle of the UI
- Each subsequent step should have “Step x” above the form fields
- State input should be a Select component with the 50 states
- Change “Zipcode” to “Zip Code”
- Show validation error if required fields are not included and the form is submitted
- On the final step, there should be a success toast popped and the user should be taken back to the initial UI
- Add Documentation CTA like shown in Network Status” example. Link to specific docs for Multi-Step Form

### CODE REVIEW

**Comment by brockinit** [comment]
@IsaiahHarris please provide a short screen shot video of the UX in future PRs.

File: examples/multistep-form/src/pages/Form.jsx [hunk 1]
```diff
@@ -32,11 +106,39 @@ const MultiStepForm = () => {
...
+    try {
+      if (validateForm()) {
+        await updateOfflineData("formData", [
+          { uuid: formData.uuid, ...formData, submitted: true },
+        ]);
+        navigate("/", { replace: true, state: { showToast: true } });
+        showToast(TOAST_CONFIG.SUCCESS);
+      }
+    } catch {
+      showToast(TOAST_CONFIG.ERROR);
```
Comment (brockinit) [inline examples/multistep-form/src/pages/Form.jsx:136]: Does this use the new Toast changes @thgaskell implemented?

File: examples/multistep-form/package.json [hunk 2]
```diff
@@ -4,7 +4,6 @@
   "private": true,
   "dependencies": {
     "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
-    "@nmfs-radfish/react-radfish": "^0",
```
Comment (brockinit) [inline examples/multistep-form/package.json:7]: Why was this removed?

Comment (thgaskell) [reply examples/multistep-form/package.json:7]: This should be added back. But it was removed for local development is an easy way to reference a local version of the package.
